docs/contributing.rst

```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
 
 Starting on Sept 1, 2017, CUAHSI will assume the role of the "benevolent dictator" of this production Hydroshare code base. After that point, pull requests into develop (and ultimately master) will only be merged by CUAHSI staff or other appointed staff from partner institutions. Other users can still:
 
-#) Create issues pull requests
+#) Create issues and/or pull requests
 #) Request reviews from anybody.
 
-Pull requests will not be merged until at least one code review is accepted. We may suggest some changes or improvements or alternatives. Some things that will increase the chance that your pull request is accepted:
+Pull requests will not be merged until at least one code review is accepted. Additionally, a likely-non-coder stakeholder must review the work to make sure that their issue is resolved or that it doesn't cause any additional technical or philosophical problems.
+
+We may suggest some changes or improvements or alternatives. Some things that will increase the chance that your pull request is accepted:
```

Comment:
Some things that will increase the chance that your pull request is accepted: --> Below is a list of best practices that will help expedite the process of accepting developer code into the master branch: